laborchestrator/vu_lab/wrappers/generic_robot_arm_wrapper.py

```diff
@@ -36,7 +39,7 @@ def get_SiLA_handler(
         # with intermediate actions we need to use the standard sila labware transfer feature
 
         class TransferHandler(ObservableProtocolHandler):
-            def _protocol(self, client: ArmClient, **kwargs) -> None:
+            def _protocol(self, client: ArmClient, **_kwargs: dict[str, Any]) -> None:
```

Comment:
Is the underscore in `_kwargs` because it's a nested class? Does it interfere with the outer `kwargs`?